drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -120,6 +123,17 @@ public void setChromeOptions(WebDriverConfiguration configuration, DesiredCapabi
         String browser = configuration.getBrowser().toLowerCase();
         if (browser.equals("chromeheadless")) {
             chromeOptions.addArguments("--headless");
+            if (configuration.getDimensions() != null) {
+                String dimensions = configuration.getDimensions().toLowerCase().trim();
+                Matcher m = DIMENSIONS_PATTERN.matcher(dimensions);
+                if (m.matches()) {
+                    String width = m.group(1);
+                    String height = m.group(2);
```

Comment:
the lines 126 - 131 are same as those ones in the windows resizer class - 75-81. Please extract it to not have it duplicated